app/views/content/what-is-qts.md

```diff
@@ -26,26 +31,43 @@ keywords:
 
-## How to get QTS
+$international-content$
 
-All postgraduate teacher training courses in England lead to QTS. You can see the different types of courses available by [searching for postgraduate teacher training](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+## Get QTS through teacher training
 
-Entry requirements for QTS courses include having an undergraduate degree and GCSEs at level 4/grade C or above in Maths and English. You’ll also need a level 4/grade C or above
-in Science if you want to be a primary school teacher.
+You can get QTS through undergraduate or postgraduate initial teacher training. This mostly involves school placements with some theoretical learning.
 
-As well as offering QTS, many courses also offer a postgraduate certificate in education (PGCE) or postgraduate diploma in education (PGDE). Find out about [PGCE and PGDE](/what-is-a-pgce).
+You can apply for either a primary or secondary teacher training course awarding QTS. 
 
-## Assessment only route to QTS
+[Find out more about teacher training](/train-to-be-a-teacher/initial-teacher-training).
 
-It is possible to get QTS through an [assessment only programme](/assessment-only-providers) if you have the right teaching experience. You can usually do this if:
+You can also get a postgraduate qualification through teacher training such as a [postgraduate certificate in education (PGCE)](/what-is-a-pgce). 
 
-* you’ve already been employed as an unqualified teacher for a significant amount of time (for example, 2 years) and you’ve worked in more than one school
-* [you’re a qualified teacher who trained overseas](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk)
+Some courses just offer a PGCE without QTS, but this means you may not be fully qualified to teach in your chosen school.
 
-<a href="/train-to-be-a-teacher" class="button">Discover the different ways to train</a>
+[Find out how to get QTS through teacher training](/train-to-be-a-teacher).
+
+## Get QTS through an assessment only programme
+
+You could get QTS through an assessment only programme. This means you can show that you already meet the standards for QTS without having to do any more training.
+
+You can usually do this if you’ve worked as an unqualified teacher:
+
+* for a significant amount of time (for example, 2 years)
+* in more than one school
+
+[Find out how to get QTS through assessment only](/train-to-be-a-teacher/if-you-have-experience).
+
+## Get QTS if you qualified as a teacher outside England
+
+If you trained to teach in Wales, your QTS will be automatically recognised in England and awarded by the [Education Workforce Council (EWC)](https://www.ewc.wales/site/index.php/en/).
+
+If you trained to teach in Scotland, Northern Ireland or outside the UK, you may be able to apply for QTS in England without any further fees or training.
```

Comment:
@EllieNodder Is the 'outside the UK' obvious enough? I read Scotland and Northern Ireland only when first scanning through, and then wondered why we didn't talk about non-UK people :-)

Might just be me though!

Reply:
@gemmadallmandfe 
```suggestion
If you trained to teach anywhere else outside England (including countries outside the UK), you may be able to apply for QTS without any further fees or training.
```
do you think this works better?

Reply:
How about 'If you trained to teach in Scotland, Northern Ireland or in a country outside the UK,....'